package.json

```
@@ -62,6 +63,14 @@
   },
   "browser": {
     "./lib/MessageDigest.js": "./lib/MessageDigest-browser.js",
+    "./lib/MessageDigest-reactnative.js": false,
+    "fast-text-encoding": false,
+    "isomorphic-webcrypto": false,
+    "rdf-canonize-native": false
+  },
+  "react-native": {
+    "./lib/MessageDigest.js": "./lib/MessageDigest-reactnative.js",
+    "./lib/MessageDigest-browser.js": false,
```

I think this can be simpler.  No need to mask out deps that will never be required.  The `rdf-canonize-native` one is needed since that is speculatively loaded.  (And that code pattern could be avoided too.)
```suggestion
    "rdf-canonize-native": false
  },
  "react-native": {
    "./lib/MessageDigest.js": "./lib/MessageDigest-reactnative.js",
```